Parameterize character counter in builder WYSIWYG editors (#673)

patternleaf opened the pull request:
Note, re git best practices: had to pull from piton/develop on account of updated tastypie on server. :(

patternleaf committed:
Make wysiwig character counter optional (#673).

Parameterize via flag passed to getEditor.

patternleaf committed:
Merge branch 'develop' of https://github.com/PitonFoundation/atlas into 673-character-count-ubiquitous-in-builder

patternleaf committed:
Move char limit config out of template (#673).

RichTextEditorMixin character count limit can be set as needed via
getEditor param, rather than in template source.

ghing commented:
@patternleaf Did you see [my comment](https://github.com/PitonFoundation/atlas/pull/758#commitcomment-3188085) from Friday? Let me know your thoughts.

patternleaf committed:
Refine character count limit semantics (#673).

- Remove default character count limit. Require specific count limit or
none when calling getEditor() on RichTextEditorMixin.

- Optimize site of argument validation.

patternleaf commented:
@ghing I see your points. 401a4af should address them. Let me know if it looks ok now or not.
